apps/src/code-studio/components/ReportAbuseForm.jsx

```diff
@@ -83,13 +83,13 @@ export default class ReportAbuseForm extends React.Component {
         <p>{msg.reportAbuseIntro()}</p>
         <br />
         <form action="/report_abuse" method="post">
+          <RailsAuthenticityToken />
           <input
             type="hidden"
-            name="authenticity_token"
-            value={this.props.csrfToken}
+            name="channel_id"
+            defaultValue={this.getChannelId()}
           />
-          <input type="hidden" name="channel_id" value={this.getChannelId()} />
-          <input type="hidden" name="name" value={this.props.name} />
+          <input type="hidden" name="name" defaultValue={this.props.name} />
```

Comment:
The change from using the `value` prop to the `defaultValue` prop on these two hidden fields fixes a React warning about passing a `null` value prop.  It also tripped prettier to reformat in a way that makes this diff extra-confusing.  Look carefully!